assets/content/content.json

```diff
@@ -104,8 +104,9 @@
     "propertyDetails": {
         "residentialAddress": "Residential Address",
         "testAddress": "5021 Kingsway, Burnby, V4H 4A5",
-        "titleDetails": "Title Details",
+        "propertyDetails": "Property Details",
         "titleNumber": "Title Number",
+        "titleNumberTitle": "Title number",
```

Comment:
Are keeping both the overall title number and the individual title numbers?

Reply:
I think so, in the case of multiple titles the header for each would be "Title number 1", "Title number 2", etc., And then under Property details would be the actual title number ex: "Title Number CA5006683" (these are the new designs I am referring to: https://www.figma.com/file/Qu9ANuVsglGqTnbXCrzhlt/BC-VHERS?node-id=2106%3A20337&mode=dev)

The reason why I have two versions of the content is because the letters have different casing, but I think I'll update the naming to make it a little clearer.

Reply:
Ah okay, makes sense. Thanks for the clarification